ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +49,36 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  findRelevantAlert = () => {
+    const { alertSummary } = this.props;
+    let relevantAlert = alertSummary.alerts.find((alert) => alert.status === 0);
+    // eslint-disable-next-line prefer-destructuring
+    if (relevantAlert.length > 1) relevantAlert = relevantAlert[0];
+    return relevantAlert;
+  };
+
+  // eslint-disable-next-line camelcase
+  getTaskId = async (push_id) => {
```

Comment:
Same for this function: it should provide the task ids for each alert.

Reply:
same as the comment above